translator/conventions/opencensus.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conventions
+
+// OTLP attributes to map certain OpenCensus proto fields. These fields don't have
+// corresponding fields in OTLP, nor are defined in OTLP semantic conventions.
+// TODO: decide if any of these must be in OTLP semantic conventions.
+const (
+	OcAttributeProcessStartTime  = "opencensus.starttime"
```

Comment:
nit (here and below): should be capitalized as `OC`, see https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
done